EngineLayer/ModernSearch/ModernSearchEngine.cs

```diff
@@ -70,7 +70,7 @@ protected override MetaMorpheusEngineResults RunSpecific()
                     // note that this is the OPPOSITE of the classic search (which calculates experimental masses from theoretical values)
                     // this is just PRELIMINARY precursor-mass filtering
                     // additional checks are made later to ensure that the theoretical precursor mass is acceptable
-                    var notches = MassDiffAcceptor.GetAllowedPrecursorMassIntervals(scan.PrecursorMass);
+                    var notches = MassDiffAcceptor.GetAllowedPrecursorMassIntervalsFromTheoreticalMass(scan.PrecursorMass);
```

Comment:
does this need to be `GetAllowedPrecursorMassIntervalsFromObservedMass`?

Reply:
Yes, it should be. Good catch.
Curious that this wasn't caught in the modern/classic comparison before. Perhaps no missed mono in that test?

Reply:
Ahh, The "Observed" code was just happening below it. I've simplified it now.